modules/gdnative/arvr/arvr_interface_gdnative.cpp

```diff
@@ -99,13 +98,9 @@ int ARVRInterfaceGDNative::get_capabilities() const {
 }
 
 bool ARVRInterfaceGDNative::get_anchor_detection_is_enabled() const {
-	bool enabled;
+	if (interface == NULL) return false;
```

Comment:
CC @BastiaanOlij

Reply:
Fine by me, does the same, my only question is why are we removing the ERR_FAIL_COND_V macro? Did I miss a memo or is this an arbitrary change?

Reply:
This is needed (similar to the one below) in order to not spam the console with errors while getting default values of a class.